### ISSUE TITLE
[ios][expo-facebook] only standalone ios app is crash

### ISSUE THREAD

**Issue by akichim21** [expo/expo#11563]
### Summary of Issue 
* When call Facebook.logInWithReadPermissionsAsync, standalone(testflight) ios app is crash.
  - expo client ios, standalone android app is not problem.
* Cannot handle errors in try catch

### Environment - output of `expo diagnostics` & the platform(s) you're targeting

```
  Expo CLI 4.0.8 environment info:
    System:
      OS: macOS 10.15.7
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.18.4 - /usr/local/bin/node
      npm: 6.14.6 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 14.3, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    IDEs:
      Xcode: 12.3/12C33 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~40.0.0 => 40.0.0
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: ~0.63.4 => 0.63.4
      react-native-web: ~0.13.12 => 0.13.18
    npmGlobalPackages:
      expo-cli: 4.0.8
    Expo Workflow: managed
```

```
expo-facebook: 9.1.0

ios: 14.2
```

### Reproducible Demo

```
      await Facebook.initializeAsync({
        appId: "APP_ID",
      });
      const result = await Facebook.logInWithReadPermissionsAsync({
        permissions: ["public_profile", "email"],
      });
```
 
We tried different app.json
```
facebookAppId
facebookDisplayName
facebookScheme
```

idevicesyslog
```
Jan  8 19:47:38 ReportCrash[4074] <Notice>: Launched ReportCrash to service - com.apple.ReportCrash
Jan  8 19:47:38 ReportCrash(CrashReporterSupport)[4074] <Notice>: Trying to create CR directory structure as root
Jan  8 19:47:38 SpringBoard(UserNotificationsServer)[59] <Notice>: [com.xxxx.xxxx] com.xxxx.xxxxapplication state changed to Terminated
Jan  8 19:47:38 SpringBoard(UserNotificationsServer)[59] <Notice>: [com.xxxx.xxxx] Ignore becoming background for application without push registration
Jan  8 19:47:38 runningboardd(RunningBoard)[32] <Error>: RBSStateCapture remove item called for untracked item <RBProcessMonitorObserver| qos:0 qos:(null) configs:{
        }>
Jan  8 19:47:38 runningboardd(RunningBoard)[32] <Error>: RBSStateCapture remove item called for untracked item <RBConnectionClient| 4072 name:application<com.xxxx.xxxx> entitlements:<RBEntitlements| [

                        ]> inheritanceManager:<RBClientInheritanceManager| >>
Jan  8 19:47:38 runningboardd(RunningBoard)[32] <Notice>: Calculated state for daemon<com.apple.SpringBoard>: running-active (role: UserInteractiveNonFocal)
Jan  8 19:47:38 runningboardd(RunningBoard)[32] <Error>: RBSStateCapture remove item called for untracked item 32-4072-57987 (target:[application<com.xxxx.xxxx>:4072])
Jan  8 19:47:38 runningboardd(RunningBoard)[32] <Notice>: [daemon<com.apple.SpringBoard>:59] Ignoring jetsam update because this process is not memory-managed
Jan  8 19:47:38 runningboardd(RunningBoard)[32] <Notice>: [daemon<com.apple.SpringBoard>:59] Ignoring suspend because this process is not lifecycle managed
Jan  8 19:47:38 runningboardd(RunningBoard)[32] <Notice>: [daemon<com.apple.SpringBoard>:59] Ignoring GPU update because this process is not GPU managed
Jan  8 19:47:38 homed(HomeKitDaemon)[175] <Notice>: Received app state change: <RBSProcessStateUpdate| process:[daemon<com.apple.SpringBoard>:59] oldState:<RBSProcessState| task:running-active debug:none endowments:[
        com.apple.frontboard.visibility
        ] rbAssertions:[
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"injecting inherited from "UIScene:com.apple.frontboard.systemappservices::com.apple.springboard" to 59<UIScene:com.apple.frontboard.systemappservices::com.apple.springboard>">,
        <RBSProcessAssertionInfo| type:2 reason:0 name:"Domain" domain:"com.apple.underlying:UnderlyingDarwinRoleUI" expl:"RB Underlying Assertion">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"creating 59<UIRootWindow:0x102239e40> visibility">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"injecting inherited from "UIRootWindow:0x102239e40" to 59<UIScene:com.apple.frontboard.systemappservices::com.apple.springboard>">,
        <RBSProcessAssertionInfo| type:1 reason:10005 name:"Custom" domain:"(null)" expl:"FBSystemShell">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"injecting inherited from "UIScene:com.apple.frontboard.systemappservices::com.apple.UIKit.remote-keyboard" to 59<UIHostedWindow:653deb4a>">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"FBSystemApp-PreventIdleSleep">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"creating 59<UIRootWindow:0x102235370> visibility">
        ]> newState:<RBSProcessState| task:running-active debug:none endowments:[
        com.apple.frontboard.visibility
        ] rbAssertions:[
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"injecting inherited from "UIScene:com.apple.frontboard.systemappservices::com.apple.UIKit.remote-keyboard" to 59<UIHostedWindow:653deb4a>">,
        <RBSProcessAssertionInfo| type:2 reason:0 name:"Domain" domain:"com.apple.underlying:UnderlyingDarwinRoleUI" expl:"RB Underlying Assertion">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"FBSystemApp-PreventIdleSleep">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"creating 59<UIRootWindow:0x102235370> visibility">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"creating 59<UIRootWindow:0x102239e40> visibility">,
        <RBSProcessAssertionInfo| type:1 reason:0 name:"Custom" domain:"(null)" expl:"injecting inherited from "UIRootWindow:0x102239e40" to 59<UIScene:com.apple.frontboard.systemappservices::com.apple.springboard>">,
        <RBSProcessAssertionInfo| type:1 reason:10005 name:"Custom" domain:"(null)" expl:"FBSystemShell">
        ]> exitEvent:(null)>
```

### Expected Behavior vs Actual Behavior

* display dialog VS crash

* expo client ios is not problem
![2021-01-08 19 56 50](https://user-images.githubusercontent.com/6077520/104008004-27651600-51ec-11eb-8920-c6db5df5886b.png)

* standalone android app is not problem
![Screenshot_20210108_195808_com onlystory onlystoryapp](https://user-images.githubusercontent.com/6077520/104008029-32b84180-51ec-11eb-8955-e786ca1d2132.jpg)

**Comment by joshsmith27** [comment]
I'm having the exact same issues. I've been looking through similar issues without any luck.
https://github.com/expo/expo/issues/9218 Recommendation update facebookScheme in the app.json. That didn't help
https://github.com/expo/expo/issues/11160 Recommendation update the app.json. That didn't help
https://forums.expo.io/t/facebook-login-with-expo-facebook-crashing-on-ios/41170 Recommendation remove FacebookUrlSchemeSuffix. That didn't help. 

I'm not sure what else to try.

**Comment by LucasPMM** [comment]
This application crash was happening with me on iOS and I managed to solve it by adding the GoogleService-Info.plist on the main folder. Is there any dependency between expo-facebook and GoogleService-Info.plist?

**Comment by akichim21** [comment]
App crashes on IOS 14.3 as well

**Comment by akichim21** [comment]
Hi.
This problem was solved in expo-facebook v10.0.0
Great job!
thanks.

**Comment by NikitaPh** [comment]
Hi, have same kind of problem with crashing with expo-facebook ~11.0.5
